src/components/App/App.tsx

```diff
@@ -282,18 +282,31 @@ export default class App extends React.Component<null, AppState> {
     });
     socket.on('REC:host', (data: any) => {
       let currentMedia = data.video || '';
-      if (this.isScreenShare() && !currentMedia.startsWith('screenshare://')) {
+      if (
+        this.isScreenShare() &&
+        typeof currentMedia === 'string' &&
```

Comment:
It's to avoid this extra typechecking that I think video metadata should be separate from the existing string representing the current video.